dbt/adapters/athena/impl.py

```diff
@@ -731,3 +731,24 @@ def get_columns_in_relation(self, relation: AthenaRelation) -> List[AthenaColumn
         return [
             AthenaColumn(column=c["Name"], dtype=c["Type"], table_type=table_type) for c in columns + partition_keys
         ]
+
+    @available
+    def delete_from_glue_catalog(self, relation: AthenaRelation):
+        schema_name = relation.schema
+        table_name = relation.table
```

Comment:
```suggestion
        table_name = relation.identifier
```
see this https://github.com/dbt-athena/dbt-athena/pull/273/files

Reply:
Makes sense. I've updated the code. Also made the change here:

https://github.com/dbt-athena/dbt-athena/blob/77491f200db20052fd0d1943e37bb114e4181aaa/dbt/include/athena/macros/adapters/relation.sql#L1-L6